dockstore-client/src/main/java/io/dockstore/client/cli/nested/ToolClient.java

```diff
@@ -753,7 +753,9 @@ public void handleInfo(String entryPath) {
                 errorMessage("This container is not published.", Client.COMMAND_ERROR);
             } else {
 
-                Date lastBuild = container.getLastBuild();
+                final long lastBuildLong = container.getLastBuild();
```

Comment:
See comment below; this should be Long not long, and you should do a null check.

Reply:
Richard Hansen fixed it